strongbox-testing/strongbox-testing-core/src/main/java/org/carlspring/strongbox/util/TestFileUtils.java

```diff
@@ -21,4 +30,50 @@ public static void deleteIfExists(File file)
         }
     }
 
+    public static void generateFile(ZipOutputStream zos,
+                                    long size)
+            throws IOException
+    {
+        generateFile(zos, size, "file-with-given-size");
+    }
+
+    public static void generateFile(ZipOutputStream zos,
+                                    long size,
+                                    String name)
+            throws IOException
+    {
+        ZipEntry ze = new ZipEntry(name);
+        zos.putNextEntry(ze);
+
+        RandomInputStream ris = new RandomInputStream(size);
```

Comment:
I know that you pasted in from `MavenArtifactGenerator `, but could you use here [The try-with-resources Statement](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) , please  ?